src/wp-includes/script-loader.php

```diff
@@ -1599,7 +1600,6 @@ function wp_default_styles( $styles ) {
 		'wp-jquery-ui-dialog',
 		// Package styles.
 		'wp-reset-editor-styles',
-		'wp-editor-classic-layout-styles',
```

Comment:
Nit: I think keeping this here would have been fine (register the style but don't add it to the dependencies, so it doesn't load). Maybe some folks could decide to force load it even if they have theme.json.

Reply:
reverted at https://github.com/WordPress/wordpress-develop/pull/1290/commits/590155bc45ffffb3293d5e0c93a6b723801ffebc